src/Poly.jl

```diff
@@ -2936,19 +2936,24 @@ function polynomial_to_power_sums(f::PolyRingElem{T}, n::Int=degree(f)) where T
 end
 
 @doc raw"""
+    power_sums_to_polynomial(P::Vector{T}) where T <: RingElement -> PolyRingElem{T}
     power_sums_to_polynomial(P::Vector{T};
-                     parent::PolyRing{T}=PolyRing(parent(P[1])) where T <: RingElement -> PolyRingElem{T}
+                      parent::PolyRing{T} where T <: RingElement -> PolyRingElem{T}
 
 Uses the Newton (or Newton-Girard) identities to obtain the polynomial
 with given sums of powers of roots. The list must be nonempty and contain
 `degree(f)` entries where $f$ is the polynomial to be recovered. The list
 must start with the sum of first powers of the roots.
 """
-function power_sums_to_polynomial(P::Vector{T}; 
-                           parent::PolyRing{T}=PolyRing(parent(P[1]))) where T <: RingElement
+function power_sums_to_polynomial(P::Vector{T}; parent::PolyRing{T}) where T <: RingElement
```

Comment:
This signature is identical to the one 4 lines below (up to kwargs). And since julia ignores the kwargs for dispatch, it cannot decide between the two